### DISCUSSION TITLE
Supress playing a sound during start of the ESCAM PT202

### DISCUSSION THREAD

**Discussion by GqSteve** [roleoroleo/yi-hack-Allwinner-v2#526] [Q&A]
Hi,

The ESCAM PT202 plays a sound, when the camera starts.

Is there a way to suppress this playback?

**Comment by roleoroleo** [comment]
What sound?

**Reply by GqSteve** [reply]
There is a short sound of 3-4s playing, when the camera starts

**Comment by roleoroleo** [comment]
Please, check which file it is:
[audio_file.tgz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/10011850/audio_file.tgz)

**Comment by GqSteve** [comment]
It is the common/poweron.aac sound file. 

I see actually the problem as "whole"....when the camera is hidden somewhere, then another sound might be played...because the SD card is full. Would it be possible to remove the audio_file.tgz from the firmware (extract image, remove file, and recreate the firmware)?

**Reply by roleoroleo** [reply]
These files are inside a read-only partition, so I can't delete them.
But I could "remove" overwriting with a mount bind command.
I don't know what's the behavior of the cam when it doesn't find a file or when the file is zero size and it's not possible to test all cases.
Try this file where I removed poweron.aac and let me know if the cam works and if it plays something.
Decompress it to /tmp/sd/yi-hack/script (rename previous file).
[system.sh.gz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/10022537/system.sh.gz)

**Comment by GqSteve** [comment]
I will be able to test it in a couple of week and will give you a feedback asap. Thanks!
